WordPress/Classes/System/Constants.m

```diff
@@ -24,6 +24,11 @@
 NSString *const WPBlavatarBaseURL                                   = @"http://gravatar.com/blavatar";
 NSString *const WPGravatarBaseURL                                   = @"http://gravatar.com/avatar";
 
+/// Image file names
+///
+NSString *const WPIconShowPassword                                  = @"icon-secure-text-visible";
```

Comment:
Can we use the `visible` and `not-visible` Gridicons here instead of images? Or are they not the same as the ones we're using?